xAPI.md

```diff
@@ -2488,6 +2487,15 @@ definitions for multiple identifiers. E.g. http://example.com/xapi/verbs#defenes
 
 ##### LRS Requirements
 * The LRS MAY act as a [Metadata Consumer](#def-metadata-consumer) and attempt to resolve identifier IRIs.
+* If an Activity IRI is an IRL, an LRS SHOULD attempt to GET that IRL, and include in HTTP
```

Comment:
We should avoid spreading "IRL" further. I think using URL here is fine; while the URL RFCs technically don't include all all IRIs-that-can-be-transformed-into-URLs, which is annoying, in practice the term URL has come to include those.